specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/runHistory.json

```diff
@@ -0,0 +1,3042 @@
+            "in": "path",
+            "description": "The experiment name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "queryParams",
+            "in": "body",
+            "description": "Query parameters for data sorting and filtering.",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/QueryParams"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/PaginatedRunList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/history/v1.0/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/experiments/{experimentName}/runcounts": {
+      "get": {
+        "tags": [
+          "Run"
+        ],
+        "summary": "Get Run counts.",
+        "description": "Get Run counts for an Experiment.",
+        "operationId": "Runs_GetCounts",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/workspaceNameParameter"
+          },
+          {
+            "name": "experimentName",
+            "in": "path",
+            "description": "The experiment name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "description": "Allows for filtering the collection of resources.\r\nThe expression specified is evaluated for each resource in the collection, and only items where the expression evaluates to true are included in the response.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "$continuationtoken",
+            "in": "query",
+            "description": "The continuation token to use for getting the next set of resources.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "$orderby",
+            "in": "query",
+            "description": "The list of resource properties to use for sorting the requested resources.",
+            "required": false,
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "collectionFormat": "multi",
```

Comment:
Is the format `multi` or `csv`? OData would use `csv`.

Reply:
This is the ODATA orderby syntax, so like 

?$orderby=Rating
?$orderby=Rating asc
?$orderby=Rating,Category/Name desc

Not sure what CSV has to do with this, unless that is another different acronym for the ODATA orderby syntax

Reply:
`"collectionFormat":"multi"` means that you can specify `?$orderby=Rating&$orderby=Category/Name` (multiple instances of `$orderby` parameters. Using `"collectionFormat":"csv"` means that you have a single `$orderby` query parameter with a comma-separated set of values. See [`collectionFormat`](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#fixed-fields-7) more information.

Reply:
Ok, we actually support neither of those things - our cosmosdb library does not allow multiple order by

Reply:
@andscho-msft, As we discussed offline, I will leave it as multi

Reply:
done